CHANGELOG.rst

```diff
@@ -1,7 +1,7 @@
 Changelog for Mantarray Frontend Components
 ===========================================
 
-1.2.4 (unreleased)
+1.2.4 (2022-11-3)
```

Comment:
Don't want to hold this PR up, but we should change this to 2022-11-03 in a future PR

Reply:
i'll do it for my next one